.vscode/launch.json

```diff
@@ -1,25 +1,23 @@
 {
-    // Use IntelliSense to learn about possible Node.js debug attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
-
-        {
-            "type": "node",
-            "request": "launch",
-            "name": "Launch Program",
-            "program": "${file}",
-            "outFiles": [
-                "${workspaceRoot}/dist/src/*.js"
-            ],
-            "args": ["--host", "127.0.0.1", "--port", "50051", "--requestId", "100a", "--workerId", "worker1"]
-        },
```

Comment:
Why are we removing this?

Reply:
IMO this one is pretty useless since it doesn't use a real function app and just fakes the args. I always test with the "Attach by Process ID" one as described in the README. If you find this one useful, though, we can leave it in

Reply:
No, I also just debug using the "Attach by Process ID," but I wasn't sure if maybe this was used for other purposes and removed by mistake. Thanks!